cob_bringup/tools/pc_monitor.launch

```diff
@@ -6,12 +6,12 @@
 	<arg name="diag_hostname" default="$(arg pc)"/>
 
 	<!-- Monitor CPU (temp, usage) -->
-	<node pkg="cob_monitoring" name="$(anon realtime_cpu_monitor)" type="cpu_monitor.py" args="--diag-hostname=$(arg diag_hostname)" output="screen">
+	<node pkg="cob_monitoring" name="$(anon cpu_monitor)" type="cpu_monitor.py" args="--diag-hostname=$(arg diag_hostname)" output="screen">
```

Comment:
let's get rid of `anon` here
we could change it to `name="cpu_monitor_$(arg pc)"`